polaris/polaris/management/commands/poll_pending_deposits.py

```diff
@@ -220,6 +231,14 @@ def execute_deposits(cls):
         for t in ready_multisig_transactions:
             cls.execute_deposit(t)
 
+        ready_claimable_transactions = Transaction.objects.filter(
+            kind=Transaction.KIND.deposit,
+            status=Transaction.STATUS.pending_anchor_claimable_start,
```

Comment:
As I said in another comment, we can't add a `status` value for our own internal processing, `status` must match the SEPs. If you need to be able to determine if a transaction should be sent using CBs, you need another method of doing so.

Ideally there is no change to the data model and Polaris just figures it out at runtime. Or, we could add an additional column, similar to our `pending_signatures` column, which Polaris uses to determine if a transaction needs additional signers before submitting.

Don't pick the solution that you can implement the easiest, pick the best solution in terms of design and efficiency.

Reply:
Reverted the `pending_anchor_claimable_start` status changes.